tests/e2e_regression/run.sh

```diff
@@ -110,16 +140,23 @@ for ((i = 0; i < nCases; i++)); do
   seen+=("$name")
 
   echo "Running test case: $name"
-  # Fetch the server response for $url
-  curl --silent --show-error --dump-header "$outDir/$name.headers" "$url" >"$outDir/$name.body"
-  # Try to pretty-print and normalize (for stable diffs) the output.
-  # If `jq` fails, output was probably not JSON; leave it as-is.
-  jq 'if .latest_update_age_ms? then .latest_update_age_ms="UNINTERESTING" else . end' \
-    <"$outDir/$name.body" \
-    >/tmp/pretty 2>/dev/null &&
-    cp /tmp/pretty "$outDir/$name.body" || true
-  # Sanitize the current timestamp out of the response header so that diffs are stable
-  sed -i -E 's/^(Date|Content-Length|Last-Modified): .*/\1: UNINTERESTING/g' "$outDir/$name.headers"
+  if [[ "$param" = /v1* ]]; then
```

Comment:
Maybe checking if `$name` starts with `db:` would be still hacky but a tiny bit nicer check to decide between this being an API or a DB a check.

Reply:
Hehe, my preference for "tiny bit nicer" leans the other way. I'll leave as-is as it seems it's not a strong preference on your end either. Anyway, we can easily change this later too, or introduce a more properly structured approach if needed.